internal/commands/buildpack_inspect.go

```diff
@@ -0,0 +1,67 @@
+package commands
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/buildpacks/pack"
+	"github.com/buildpacks/pack/internal/config"
+	"github.com/buildpacks/pack/internal/style"
+	"github.com/buildpacks/pack/logging"
+)
+
+type BuildpackInspectFlags struct {
+	Depth    int
+	Registry string
+	Verbose  bool
+}
+
+func BuildpackInspect(logger logging.Logger, cfg config.Config, client PackClient) *cobra.Command {
+	var flags BuildpackInspectFlags
+	cmd := &cobra.Command{
+		Use:     "inspect <image-name>",
+		Args:    cobra.MaximumNArgs(1),
+		Short:   "Show information about a buildpack",
+		Example: "pack buildpack inspect cnbs/sample-package:hello-universe",
+		RunE: logError(logger, func(cmd *cobra.Command, args []string) error {
+			buildpackName := args[0]
+			registry := flags.Registry
+			if registry == "" {
+				registry = cfg.DefaultRegistryName
```

Comment:
This is a slight change of behavior – in `inspect-builder`, it checks the configs default-registry, rather than default-registry-name, but given that `default-registry` is deprecated, I thought it appropriate to use the newer `default-registry-name` for the updated command.